### DISCUSSION TITLE
hi, i want to make it hot reload enabled

### DISCUSSION THREAD

**Discussion by plugeit** [SKbarbon/Flet_StoryBoard#11] [Q&A]
The problem is i have to click done button but using motion and keystroke listener i want to run it each time any keystrokes or mouse clicks are registered so for that reason i was trying to modify the code so i searched for done button in the source code and i found it in edit_subwidgets_engine.py file but when i tried to change the value of button nothing happened like i tryed changing the text in done button to "testing" but it didn't work

**Comment by SKbarbon** [comment]
Thank you @icanhackyou for using flet storyboard. Please wait the v2.0 this where this issue is fixed.
